samples/net/sockets/raw/README.rst

```diff
@@ -0,0 +1,27 @@
+.. _raw-socket-sample:
+
+Raw socket sample
+#################
+
+Overview
+********
+
+This sample is a simple RAW socket application showing usage of
+raw socket over ethernet. Sample prints every packet which
+etherent receives and sends every packet over the wire. Zephyr
+network subsystem does not touch any of the headers(L2, L3 and
+so on).
+
+Building and Running
+********************
+
+When the application runs, it opens a raw socket and prints
+the length of packet it receives. And sends dummy packet over
```

Comment:
So. it sends not "every packet" after all, but a dummy one every 5s. Re: https://github.com/zephyrproject-rtos/zephyr/pull/12564/files#r251889978